templates/taxi/car_detail.html

```diff
@@ -10,6 +10,17 @@ <h1>
     <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
       Update
     </a>
+    {% if user_is_driver %}
+      <form action="{% url 'taxi:car-assign' pk=car.pk %}" method="post">
+        {% csrf_token %}
+        <button class="btn btn-danger link-to-page" type="submit">Delete me from this car</button>
+      </form>
+    {% else %}
```

Comment:
```suggestion
      <form action="{% url 'taxi:car-assign' pk=car.pk %}" method="post">
        {% csrf_token %}
         {% if user_is_driver %}
        <button class="btn btn-danger link-to-page" type="submit">Delete me from this car</button>
          {% else %}
          ........
          {% endif %}
      </form>
```
No need to duplicate code